packages/@uppy/companion/src/server/controllers/index.js

```diff
@@ -1,14 +1,27 @@
-module.exports = {
-  callback: require('./callback'),
-  deauthorizationCallback: require('./deauth-callback'),
-  sendToken: require('./send-token'),
-  get: require('./get'),
-  thumbnail: require('./thumbnail'),
-  list: require('./list'),
-  simpleAuth: require('./simple-auth'),
-  logout: require('./logout'),
-  connect: require('./connect'),
-  preauth: require('./preauth'),
-  redirect: require('./oauth-redirect'),
-  refreshToken: require('./refresh-token'),
+import callback from './callback.js'
+import connect from './connect.js'
+import deauthorizationCallback from './deauth-callback.js'
+import get from './get.js'
+import list from './list.js'
+import logout from './logout.js'
+import redirect from './oauth-redirect.js'
+import preauth from './preauth.js'
+import refreshToken from './refresh-token.js'
+import sendToken from './send-token.js'
+import simpleAuth from './simple-auth.js'
+import thumbnail from './thumbnail.js'
+
+export {
```

Comment:
You can just do `export { default as thumbnail } from './thumbnail.js'`, no need to import then export again.